src/test/java/Homework20.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework20 extends BaseTest{
+    @Test
+    public void deletePlaylist() {
+        logIn();
+        clickOnPlaylist();
+        clickOnDeletePlaylist();
+        assertPlaylistIsDeleted();
+
+    }
+
+    private void assertPlaylistIsDeleted() {
+        //Assert Delete Popup shows
+        WebElement playlistDeletedPopup = wait.until(ExpectedConditions.visibilityOfElementLocated
+                (By.xpath("//div[@class='success show']")));
+        Assert.assertTrue(playlistDeletedPopup.getText().contains("Deleted playlist"));
```

Comment:
SoftAssert asert=new SoftAssert();
    asert.assertTrue(playlistDeletedPopup.getText().contains("Deleted playlist"));
      asert.assertTrue(wait.until(ExpectedConditions.invisibilityOfElementLocated(By.xpath("//a[text()='Salma']"))));
   asert.assertAll();

Reply:
Thank you, but soft assert was not mentioned in the classes. I will look it up